### PR TITLE
Adoption removal

### LINKED ISSUE #283: Adoptions still being solicited in Wildbooks
**Open questions**
We need to preserve the adoption names that exist and credit those adopters, but remove the system charges. We should provide an alternative way for users to provide donations. One-time email alert is a good idea.

**Context**
Adoptions as they stand are not stable and have a negative impact on Wild Me image with citizen scientists in particular.

- Wildbook sends confirmation emails that prompt users to adopt their identified animal. However, adoptions are supposed to be disabled due to some payment processing issues.

- Sharkbook prompts users on the individual page to nickname their shark, which opens an adoption payment page (https://community.wildme.org/t/unable-to-name-individuals/1858). 
- MantaMatcher user attempts to name manta and gets access denied error when they're prompted for adoption (https://community.wildme.org/t/access-denied-when-submitting-adoption/1955)
- Multiple users have stated they paid for an adoption but do not receive a) payment confirmation or b) acknowledgment in Wildbook that they are the adopter of the desired individual. (https://community.wildme.org/t/not-able-to-adoption-a-specific-individual/1781) 
(email from Carol at SAS-USA to Tanya and Anastasia Jan 30, 2021)
- When you submit an encounter to [Spot-a-Shark USA](https://ncaquariums.wildbook.org/submit.jsp), the footer of the confirmation page says "You can adopt a sand tiger shark and give it a nickname. To search for adoptable sharks and select one or more sharks to adopt, click the button below."

Image

### PR DESCRIPTION
**Pull request by naknomum**
fixes #283 

- removes Adoption java class
- removes properties files (settings & lang) related to adoption
- removes various methods from other classes where related to adoption
- removes donation-related code (stripe, donorbox)
- removes references and links from jsp, email templates

### CODE REVIEW

File: src/main/java/org/ecocean/servlet/importer/FixStandardImport.java
Comment (TanyaStere42) [inline src/main/java/org/ecocean/servlet/importer/FixStandardImport.java:1]: can't we just get rid of this?

**Review by TanyaStere42 — CHANGES_REQUESTED** [review]
one little group of comments I think you can remove. doing a quick addition to adjust some copy, then this should be good to go

**Review by TanyaStere42 — APPROVED** [review]
looks good to me!

**Comment by goddesswarship** [comment]
👏

**Pull request merged**